libs/barista-components/switch/src/switch.scss

```
@@ -3,7 +3,50 @@
 @import '../../core/src/style/transition';
 @import '../../core/src/theming/theming';
 @import '../../style/font-mixins';
-@import './switch-theme';
+
+:host {
+  --content-color: #{$default-font-color};
+  --dt-switch-bar-background-radial: #{$gray-700};
+  --dt-switch-bar-border-color: #{$turquoise-600};
+  --dt-switch-bar-bg-border-color: transparent;
```

Please keep abbreviations consistent. You are sometimes using `bg` and sometimes `background`. 
I understand that some of these variables could get quite long, I would opt for `background`.
```suggestion
  --dt-switch-bar-background-border-color: transparent;
```

---

Yes, I was thinking about that during implementation, but now I think ```background``` is better.